e2e/tests/eth_integration.rs

```diff
@@ -0,0 +1,273 @@
+use contracts::{IUniswapLikeRouter, WETH9};
+use ethcontract::prelude::{Account, Address, PrivateKey, U256};
+use hex_literal::hex;
+use model::{
+    order::{OrderBuilder, OrderKind, BUY_ETH_ADDRESS},
+    SigningScheme,
+};
+use secp256k1::SecretKey;
+use serde_json::json;
+use shared::{amm_pair_provider::UniswapPairProvider, Web3};
+use solver::{
+    liquidity::uniswap::UniswapLikeLiquidity, liquidity_collector::LiquidityCollector,
+    metrics::NoopMetrics,
+};
+use std::{collections::HashSet, sync::Arc, time::Duration};
+use web3::signing::SecretKeyRef;
+
+mod ganache;
+#[macro_use]
+mod services;
+use crate::services::{
+    create_orderbook_api, deploy_mintable_token, to_wei, GPv2, OrderbookServices, UniswapContracts,
+    API_HOST,
+};
+
+const TRADER_BUY_ETH_A_PK: [u8; 32] =
+    hex!("0000000000000000000000000000000000000000000000000000000000000001");
+const TRADER_BUY_ETH_B_PK: [u8; 32] =
+    hex!("0000000000000000000000000000000000000000000000000000000000000002");
+const TRADER_SELL_ETH_PK: [u8; 32] =
+    hex!("0000000000000000000000000000000000000000000000000000000000000003");
```

Comment:
Could use fewer characters if we just use `[1u8; 32], [2u8; 32], [3u8; 32]` although they are not the exact same strings in the end.